src/wallet/rpcwallet.cpp

```diff
@@ -2389,9 +2451,9 @@ static UniValue getwalletinfo(const JSONRPCRequest& request)
             "{\n"
             "  \"walletname\": xxxxx,               (string) the wallet name\n"
             "  \"walletversion\": xxxxx,            (numeric) the wallet version\n"
-            "  \"balance\": xxxxxxx,                (numeric) the total confirmed balance of the wallet in " + CURRENCY_UNIT + "\n"
-            "  \"unconfirmed_balance\": xxx,        (numeric) the total unconfirmed balance of the wallet in " + CURRENCY_UNIT + "\n"
-            "  \"immature_balance\": xxxxxx,        (numeric) the total immature balance of the wallet in " + CURRENCY_UNIT + "\n"
+            "  \"balance\": xxxxxxx,                (numeric) Identical to getbalances().mine.trusted\n"
```

Comment:
Maybe `Deprecated, use getbalances().mine.trusted` instead?

Reply:
I agree this could be changed. My suggestion earlier was something like "Alias for getbalances().mine.trusted for backwards compatibility. This filed may be removed in a future version."

Reply:
We have a track record of removing RPC features when users relied on them, so I'd rather keep them around redundantly instead of getting yelled at again and have users stick to EOL software instead of upgrading to a recent version.

Reply:
@MarcoFalke agree it shouldn't be removed, but marking deprecated encourages using `getbalances` instead.

Reply:
Done